cmd/rookflex/cmd/mount.go

```diff
@@ -190,7 +192,9 @@ func mount(client *rpc.Client, mounter *k8smount.SafeFormatAndMount, globalVolum
 						return fmt.Errorf("%s is still mounted, despite call to unmount().  Will try again next sync loop", opts.MountDir)
 					}
 				}
-				os.Remove(opts.MountDir)
+				if err := os.Remove(opts.MountDir); err != nil {
+					return errors.Wrap(err, "failed to remove dir")
```

Comment:
let's call `log()` for this one as well so the original error can be returned on the next line